conf/osp-director.openstack.org_openstackbackuprequests.yaml

```diff
@@ -0,0 +1,147 @@
+
```

Comment:
conf is the new CRD dir from https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/Makefile#L79